AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt

```diff
@@ -1241,9 +1242,10 @@ open class CardBrowser :
     fun switchCardOrNote(newCardsMode: bool) {
         val sharedPrefs = AnkiDroidApp.getSharedPrefs(this)
 
-        sharedPrefs.edit()
-            .putBoolean("inCardsMode", newCardsMode)
-            .apply()
+        sharedPrefs.edit {
+            this.putBoolean("inCardsMode", newCardsMode)
+            this.apply()
+        }
```

Comment:
`edit` calls `apply` at the end of the call

```suggestion
        sharedPrefs.edit { putBoolean("inCardsMode", newCardsMode) }
```

Reply:
Welp that was dumb of me 😅

Reply:
Don't beat yourself up - the point of code reviews is knowledge transfer, this should encourage learning!